docs/architecture/modernize-desktop/toc.yml

```
@@ -0,0 +1,5 @@
+- name: Modernizing Desktop Apps on Windows 10 with .NET Core 3
```

```suggestion
- name: Modernizing desktop apps on Windows 10 with .NET Core 3
```

I'm not sure if ebooks have different style rules, but for non-ebook docs, we'd change this to "Modernize" instead of "Modernizing".

---

I don't think we have defined rules but we've been using "Architecting.." "Modernizing.." etc in the e-Books. If that needs to be changed, we may have to rework all of them and change the graphics as well.